### DISCUSSION TITLE
Demo website for data

### DISCUSSION THREAD

**Discussion by osmanraifgunes** [iptv-org/community#157] [Show and tell]
Hello,

I created a website using data in this repo. https://worldtvmobile.com

It will be upto date at least one year. I can open source code also if you desire.

You can use this as a demo ling at readme of this repo.

**Comment by sguinetti** [comment]
Thank you. It's interesting this web site to see in an orderly way the transmissions of channels in the world. I will add in this list repository (https://github.com/iptv-org/awesome-iptv) for users to visit the website you published.
